### PR TITLE
SelectBox: update the 'fieldTemplate' property

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md [hunk 1]
```diff
@@ -17,10 +17,106 @@ The selected item's data.
 A template name or container.
 
 ---
+If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, manually render hidden input with the corresponding name attribute to submit values through HTML form.
+
+The example below shows how to define the **fieldTemplate**. If you want to customize field text, use the TextBox [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```
Comment (AnnaKondratova) [inline api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md]:
```suggestion
The example below shows how to define the **fieldTemplate**. If you want to customize field text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```

File: api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md [hunk 2]
```diff
@@ -17,10 +17,106 @@ The selected item's data.
 A template name or container.
 
 ---
+If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, manually render hidden input with the corresponding name attribute to submit values through HTML form.
```
Comment (AnnaKondratova) [inline api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md]:
```suggestion
If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, manually render hidden input with the corresponding name attribute to submit values through a HTML form.
```

File: api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md [hunk 3]
```diff
@@ -17,10 +17,106 @@ The selected item's data.
 A template name or container.
 
 ---
+If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, manually render hidden input with the corresponding name attribute to submit values through a HTML form.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md]:
```suggestion
If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, you should render hidden input with the corresponding name attribute to submit values through a HTML form.
```

File: api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md [hunk 4]
```diff
@@ -17,10 +17,106 @@ The selected item's data.
...
+
+The example below shows how to define the **fieldTemplate**. If you want to customize field text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md]: `to customize field text` A field? or fields in general?

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md (hunk 4 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md]: A component (SelectBox or TagBox) has a field, I'm talking about such components in general.

Comment (dirk-pieterse) [reply api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md]:
```suggestion
The example below shows how to define the **fieldTemplate**. If you want to customize this field's text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```

File: api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md [hunk 5]
```diff
@@ -17,10 +17,106 @@ The selected item's data.
 A template name or container.
 
 ---
+If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, you should render hidden input with the corresponding name attribute to submit values through a HTML form.
+
+The example below shows how to define the **fieldTemplate**. If you want to customize this field's text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md]:
```suggestion
The example below shows how to define the **fieldTemplate**. If you want to customize the field's text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```

**Pull request merged**